src/Servers/Kestrel/Core/src/Internal/Infrastructure/HttpUtilities.cs

```diff
@@ -431,11 +426,12 @@ public static string VersionToString(HttpVersion httpVersion)
         {
             return httpVersion switch
             {
-                HttpVersion.Http10 => Http10Version,
-                HttpVersion.Http11 => Http11Version,
+                HttpVersion.Http10 => AspNetCore.Http.HttpProtocol.Http10,
+                HttpVersion.Http11 => AspNetCore.Http.HttpProtocol.Http11,
```

Comment:
If the other values of the `HttpVersion`-enum aren't needed and return `null`, should we add a `Debug.Assert` for this?
(I.e. to catch it early, instead of having nasty `null`-hunting in the case of).